2016/july/raising_money_for_your_(first)_python_workshop_with_corporate_sponsorships.md

```diff
@@ -0,0 +1,119 @@
+
+### What should I include in my email asking for sponsorship?
+
+Depending on the company you reach out to they may want more or less information. Some questions to ask yourself about how to frame your request:
+
+* Does the company have a community outreach team?
+* Has another organization worked with the company before?
+* Does the company already participate in the Python community?
+
+If a company has a community outreach team that means they're more experienced in working in the local tech scene but they may also receive lots of requests. Reach out to the team and ask them how they process requests. 
+
+If you know another organizer that's worked with the company in question - great! Personal introductions establish a repoire and often helps you cut velvet rope. You also get first hand knowledge from the organizer how to chat with the company. 
+
+If you don't have access to this information start simple by creating a budget and based on the big items such as food or swag you can offer sponsorship for those items or have generic sponsorship levels. Bigger events merit the latter approach whereas the former is more than sufficient for smaller events. 
+
+Back to our PyLadies Chicago event we only needed a food and venue sponsor.
+
+### When should you stop fundraising? How much funding is enough?
+
+Again this depends on the scope of the event. Organizing a 2 hour panel with 50 attendees is significantly different than organizing a one day training workshop for 50 people. You'll want to plan to have a bigger cushion. 
+
+**TODO**: Don't be too afraid of taking in too much money because you can use extra funding for future events. It's better to have too money leftover than to have your organizers lose money on a workshop.
+
+**TODO**: Link to how to create a budget, perhaps more discussion?
+
+For creating a detailed budget read this thing
+
+Generally having extra isn't a terrible thing, depending on how you raise funds (receiving monies or having sponsor pay directly) you can have spare for another event. 
+
+Be mindful of how much you ask for, you may want to work with the company again! 
+
+And as always ask for recommendations!
+
+There are several Python communities you can reach out to including:
+
+* [PSF Community Listserv](mailto:PSF-community@python.org)
+* [PyLadies Slack](slackin.pyladies.com/)
+* [IRC Python channels](https://www.python.org/community/irc/)
+* Twitter!
+
+**TODO**: More and/or better resources?
+
+### Do you need to tell potential sponsors about the other sponsors you'll have?
+
+**TODO**
+
+### A company committed but hasn't paid us yet. What should I do?
+
+**TODO**
```

Comment:
I've been fortunate enough to not have been in this situation myself we may need some outside counsel on this